reusable-workflows/README.md

```diff
@@ -51,6 +51,7 @@ options when using the workflow.
 | `AWS_ACCESS_KEY_ID_FRONTEND_REGISTRY`     | ID of a AWS key that allows r/w access to the registry bucket     |   yes    |
 | `AWS_SECRET_ACCESS_KEY_FRONTEND_REGISTRY` | Secret of a AWS key that allows r/w access to the registry bucket |   yes    |
 | `GH_REGISTRY_NPM_TOKEN`                   | Token for NPM package registry                                    |   yes    |
+| `DD_API_KEY`                              | API key for Datadog                                               |   yes    |
```

Comment:
Maybe we could add what we're using the API_KEY for here?

Reply:
Updated here https://github.com/pleo-io/spa-tools/pull/179/commits/fe576540b7fdd473f02e3685a2b14d9ddbac7c02